python/model/Percentage-Alert/main.py

```diff
@@ -0,0 +1,50 @@
+from quixstreaming import QuixStreamingClient, StreamEndType, StreamReader
+from quixstreaming.app import App
+from percentage_function import PercentageAlert
+import os
+
+
+# Create a client. The client helps you to create input reader or output writer for specified topic.
+client = QuixStreamingClient('eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IlpXeUJqWTgzcXotZW1pUlZDd1I4dyJ9.eyJodHRwczovL3F1aXguYWkvb3JnX2lkIjoicXVpeGRldiIsImh0dHBzOi8vcXVpeC5haS9yb2xlcyI6IiIsImlzcyI6Imh0dHBzOi8vYXV0aC5kZXYucXVpeC5haS8iLCJzdWIiOiJhdXRoMHw2YjljOGI1YS1kNGY0LTQzMDUtOTdlYS1hYzQ2ZmI2OGUzODUiLCJhdWQiOlsiaHR0cHM6Ly9wb3J0YWwtYXBpLmRldi5xdWl4LmFpLyIsImh0dHBzOi8vcXVpeC1kZXYuZXUuYXV0aDAuY29tL3VzZXJpbmZvIl0sImlhdCI6MTY0MTkwMTg5NCwiZXhwIjoxNjQ0NDkzODk0LCJhenAiOiI2MDRBOXE1Vm9jWW92b05Qb01panVlVVpjRUhJY2xNcyIsInNjb3BlIjoib3BlbmlkIHByb2ZpbGUgZW1haWwiLCJwZXJtaXNzaW9ucyI6W119.vpqLcoz023O3ZlddoD1Gcglm-j-btFmphrt_ur4wCMoejRo8tqbOTIDBkFRBlaL0AiduISoNBibevAu58Zx_riRoZf-KiZacehNSZHEBsk3qeiLPowEPCzJUFgiZW-LcHly-rfMzoLellrp7FKTcfu5Hg9hnkAHbWGKFeczjblElRuL4pLPcHiwNNwXlIHxm0KzegFgad7-JMKzdiKCwYruNINS83Ez0HGbsXbpDsri_0WGmXPYeX8yIbhimW0ZnQ3ZAaTwln22E87YbhE3bj4zzjHidBqIOXvdkbe8DRquPjdaEL8kvTp0Fm2Np2JKSagOteuQ4z5lLBMaj91g8Gw')
+# temporary (needed for dev)
+client.api_url = "https://portal-api.dev.quix.ai"
+
+# Change consumer group to a different constant if you want to run model locally.
+print("Opening input and output topics")
+
+# Define environmental variables
+input_topic = client.open_input_topic(os.environ["Quix__Workspace__Id"]+"-"+os.environ["input"], "default-consumer-group")
+output_topic = client.open_output_topic(os.environ["Quix__Workspace__Id"]+"-"+os.environ["output"])
+parameter_name = str(os.environ["ParameterName"])
```

Comment:
Move ParameterName and percentage to inner class ctor please so we explain how we got it.

Reply:
Do it for other projects as well, please.